packages/toggle/style.ts

```diff
@@ -0,0 +1,6 @@
+import { css } from "emotion";
+
+export const style = css`
+  display: inline-block;
```

Comment:
I think we should let component consumers decide what the `display` is. Let's _just_ make this component responsible for toggling itself on and off.

Reply:
Sounds good.